pkgs/applications/editors/gobby/default.nix

```diff
@@ -6,7 +6,7 @@ let
   libinf = libinfinity.override { gtkWidgets = true; inherit avahiSupport; };
 in stdenv.mkDerivation {
   pname = "gobby";
-  version = "unstable-2020-12-29";
+  version = "0.6.0";
```

Comment:
I think rev in fetchFromGitHub should be replaced with version.

Reply:
Oh, I missed staging that change, will push in a second.